test/DynamoCoreWpfTests/ConverterViewModelTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Controls;
+using CoreNodeModels;
+using CoreNodeModels.Input;
+using CoreNodeModelsWpf.Controls;
+using Dynamo.Tests;
+using Dynamo.Utilities;
+using DynamoCoreWpfTests;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace CoreNodeModelsWpf
+{
+    [TestFixture]
+    public class ConverterViewModelTests : DynamoTestUIBase
+    {
+        private AssemblyHelper assemblyHelper;
+
+        public override void Open(string path)
+        {
+            base.Open(path);
+
+            DispatcherUtil.DoEvents();
+        }
+
+        public override void Run()
+        {
+            base.Run();
+
+            DispatcherUtil.DoEvents();
+        }
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            SetupTests();
+            libraries.Add("VMDataBridge.dll");
+            libraries.Add("DSCoreNodes.dll");
+            libraries.Add("DynamoConversions.dll");
+            libraries.Add("DynamoUnits.dll");
+            libraries.Add("CoreNodeModelsWpf.dll");
```

Comment:
CoreNodeModelsWpf cannot be imported into the VM. I thought I had warned against doing this @RobertGlobant20. Please change.

Reply:
Also please don't call `SetupTests` inside of `GetLibrariesToPreload`. This method should purely be used to preload libraries.

Reply:
@aparajit-pratap I think you mean overwrite the existing `Start` function from base?

Reply:
@aparajit-pratap  
> CoreNodeModelsWpf cannot be imported into the VM. I thought I had warned against doing this @RobertGlobant20. Please change.

Related to this comment, I remember that you told me that I cannot add the dll in this way libraries.Add("CoreNodeModelsWpf.dll"); but I realized that I was testing UI features and if I don't add the "CoreNodeModelsWpf.dll" the next line returns null (then is not possible to get the DynamoConverterControl):

 var dynamoConverterControl = nodeView.grid.ChildrenOfType<DynamoConverterControl>().FirstOrDefault();

If is not a good idea adding the dll in this way, do you have any idea how can I add/load the dll?

cc: @QilongTang

Reply:
The dll should be loaded by the resolve assembly logic, not this. This list of dll's are for loading them into the Dynamo engine as nodes.